### DISCUSSION TITLE
math: sum in fracs rendering messed up

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#39432] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### MWE

````markdown
Sum with limits in fracs is broken. Standalone:
```math
\sum_{i=1}^n
```

In frac:
```math
\frac{\sum_{i=1}^n}{2}
```
````

##### Rendering

Sum with limits in fracs is broken. Standalone:
```math
\sum_{i=1}^n
```

In frac:
```math
\frac{\sum_{i=1}^n}{2}
```

---
Screenshot (Nov 18, 2022):

![screenshot](https://user-images.githubusercontent.com/181628/202734689-5e1c77b9-cac3-4682-bfbe-ccf4b2a6ede0.png)
---

**Comment by dpvc** [comment]
This turns out to be due to the fact that GitHub is removing some attributes from the MathJax output. In this case, the limits="false" attribute has been removed from the HTML nodes that enclose the sum character, so the needed CSS to style those character properly is not being applied.

**Comment by dmbuil** [comment]
Any news on this? fracs are still rendered funny. It is difficult to follow in complex formulas :(

**Reply by nschloe** [reply]
No, still the same. You can check [here](https://github.com/nschloe/github-math-bugs#sumproduct-signs-in-inline-mode-or-fractions) to see if the bug has been fixed. There seems to be little movement on GitHub's math/markdown front though.

**Comment by mayer79** [comment]
Quite annoying. The problem seems to be still here.

**Comment by nschloe** [comment]
Dup of #17051.
